src/main/java/ladder/domain/GameResult.java

```diff
@@ -0,0 +1,39 @@
+package ladder.domain;
+
+import java.util.LinkedHashMap;
+import java.util.Map;
+import ladder.domain.participant.Name;
+import ladder.domain.participant.Participants;
+
+public class GameResult {
+
+    private final Map<Name, String> gameResult;
+
+    public GameResult(Participants participants, Prizes sortedPrizes) {
+        this.gameResult = makeLadderGameResult(participants, sortedPrizes);
+    }
+
+    private Map<Name, String> makeLadderGameResult(Participants participants, Prizes sortedPrizes) {
+        Map<Name, String> gameResult = new LinkedHashMap<>();
+
+        for (int index = 0; index < participants.getCount(); index++) {
+            gameResult.put(participants.getNameByIndex(index), sortedPrizes.getNameByIndex(index));
+        }
+
+        return gameResult;
+    }
+
+    public void containsName(String nameInput) {
```

Comment:
Name으로 받아도 되지 않았을까요?
```suggestion
    public void containsName(Name name) {
```